### DISCUSSION TITLE
sync failed: The keys are out of order: marker "",

### DISCUSSION THREAD

**Discussion by cnglen** [juicedata/juicefs#3067] [Q&A] [answered]
juicefs sync obs://xxx/ ~/xxxx

2022/12/06 20:57:07.697654 juicefs[20209] <ERROR>: The keys are out of order: marker "", last "label_dt=20221123/" current "label_dt=20220501/feature_dt=20220430/" [sync.go:132]
2022/12/06 20:57:07.697731 juicefs[20209] <ERROR>: Listing failed, stop syncing, waiting for pending ones [sync.go:656]

**Comment by zhijian-pro** [comment, marked as answer]
We require that the listobject returned from the object store be ordered.
you can use [objbench](https://juicefs.com/docs/community/command_reference/#juicefs-objbench) command test it.
